docs/solidity-by-example.rst

```diff
@@ -126,7 +126,7 @@ of votes.
             // modifies `voters[msg.sender].voted`
             sender.voted = true;
             sender.delegate = to;
-            Voter delegate = voters[to];
+            Voter storage delegate = voters[to];
```

Comment:
Hi all,

Adding the storage keyword does not solve in Mist browser 0.9.0.
I added the storage keyword so the simplifid code below:

pragma solidity ^0.4.13;
contract Example{
    struct Student{
	    string storage studentName;
	    bool active;
    }
}

And I get: Expected identifier, got 'eth_compileSolidity' string storage studentName;

In Remix it is fine without the storage keyword.

If I am not in the right place to put this comment, please let me know.
Jan

Reply:
There was a problem in an earlier Solidity version which would report a warning on struct definitions. This can only be fixed by upgrading the compiler, and not by modifying the source code.